I/InverseDistanceWeighting/Compat.toml

```diff
@@ -26,10 +26,15 @@ StaticArrays = "0.8.3-0"
 julia = "0.7-1"
 
 ["0.2.1-0"]
+julia = "1"
+
+["0.2.1-0.2.2"]
 Distances = "0.7.3-*"
 GeoStatsBase = "0.4.2-*"
 GeoStatsDevTools = "0.4.3-*"
 NearestNeighbors = "0.4.1-*"
 Reexport = "0.2.0-*"
 StaticArrays = "0.8.3-*"
```

Comment:
Please keep

Reply:
I am very confused about these numbers in brackets. I don't know what they are supposed to mean. I only touch Project.toml files.

Reply:
Yes, but you have a `[compat]` section there where you add compatibility constraints.